route/main.go

```diff
@@ -63,6 +66,30 @@ func RegisterRoute(h *server.Hertz) {
 	})
 }
 
+func RegisterRouteWithHandle(h *server.Hertz) {
+	h.Handle(consts.MethodGet, "/hget", func(ctx context.Context, c *app.RequestContext) {
+		c.String(consts.StatusOK, "get")
```

Comment:
The content should be aligned with the method.

Reply:
done, thx